lib/utilities.js

```diff
@@ -13,6 +13,26 @@ context ${context}`);
   }
 };
 
+// safely set a value nested deeply in an objects
+export const safeSet = (obj, dotpath, data) => {
```

Comment:
This makes a lot more sense to me now looking at the code. Simple and elegant. Very cool!

Reply:
Should I add a better comment?

Reply:
I think this is fine. Just wasn't as clear to me when talking about it in words.